src/index.js

```diff
@@ -1,18 +1,9 @@
 import Exporter from './exporter';
 import createTemplate from './template';
 
-let sql;
-
-if (process.env.APP_ENV === 'browser' || typeof window !== 'undefined') {
-  require('script-loader!sql.js');
-  sql = window.SQL;
-} else {
-  sql = require('sql.js');
-}
-
 export { Exporter };
 
-export default function(deckName, template) {
+export default function(deckName, template, sql) {
```

Comment:
Should we throw an explicit error if sql argument is not defined? It would help people migrating from older code as it would fail loudly and clearly. 

I agree with @yakhinvadim, this does seem awkward as `template` is optional. I do see an advantage for using object notation for method params. This could be good pattern as other methods could benefit from this too, e.g. `addCard`.